src/app/retrospective/retrospect-task-modal/retrospect-task-modal.component.ts

```diff
@@ -498,4 +515,15 @@ export class RetrospectTaskModalComponent implements OnDestroy, AfterViewChecked
             this.gridApi.onFilterChanged();
         }
     }
+
+    setColumnData() {
```

Comment:
What does this do? As per my understanding, we are saving the column state here. If so, please rename it to `saveColumnState`. Same for other components and filter service method name.

Reply:
changed